src/distilabel/pipeline/base.py

```diff
@@ -34,6 +34,7 @@
 from typing_extensions import Self
 
 from distilabel import __version__
+from distilabel.mixins.graph_viz import GraphvizMixin
```

Comment:
Even though `graphviz` has been defined as optional, when importing it here we make it mandatory and the user will need to install it even if not planning to use it, so maybe then we can make the dependency mandatory? Otherwise we can just define a function to do so instead of a `MixIn`?

Reply:
Totally! Will take a look, maybe it's better to avoid adding the dependency, so it can be kept in a function for simplicity.